deploy/bootstrap/settings.rc

```diff
@@ -40,3 +40,9 @@ KEYS_PASSWORD=$(generate_password 32)
 if [[ -z ${CEGA_CONNECTION+x} ]] && [[ -z ${CEGA_USERS_CREDS+x} ]]; then
     source ${HERE}/settings.rc.private
 fi
+
+MQ_USER=guest
+MQ_PASSWORD_HASH=4tHURqDiZzypw0NTvoHhpn8/MMgONWonWxgRZ4NXgR8nZRBz
```

Comment:
The user can still remain `guest` but you can make use of `generate_password` in `deploy/bootstrap/defs.sh` to generate a password and then make use of `extras/rabbitmq_pwhash.py` to hash it.

the you can also update `MQ_CONNECTION = amqp://guest:guest@mq:5672/%2F` in `run.sh`

Reply:
while trying to solve this I trying to load the Local broker with a custom password and it fails with this error - this is an issue related to `LocalEGA-mq`:
```
=WARNING REPORT==== 22-Mar-2019::19:45:28 ===
HTTP access denied: user 'guest' - invalid credentials
*** Access refused: /api/definitions
```
commenting out the `640` permissions seems to partially fix this, also this was solved in helm charts like this this might help: https://github.com/NBISweden/LocalEGA-helm/blob/master/ega-charts/localega/conf/mq.sh#L10-L18

Reply:
Maybe i missed something but  for https://github.com/EGA-archive/LocalEGA/tree/feature/external-images_random-mq-pass tests fail on my computer even though i pruned images and tried different scenarios.  I assume the underlying cause is the new `lega-mq` image as the federation and shovelling don't see to work with a custom password

Reply:
[Issue resolved](https://github.com/EGA-archive/LocalEGA-mq/issues/2)

Reply:
I can confirm that `Localega-mq` can be started with custom password for the latest commits.
However, I need to set both `MQ_PASSWORD_HASH` and `MQ_PASSWORD` to start the `lega-mq` service. That is

```
docker run -e MQ_USER=admin  -e MQ_PASSWORD_HASH=$MQ_PASSWORD_HASH -e MQ_PASSWORD=$MQ_PASSWORD -e CEGA_CONNECTION=amqp://$MQ_USER:$MQ_PASSWORD@cega-mq:5672/lega egarchive/lega-mq
```